requirements.txt

```diff
@@ -1,5 +1,3 @@
-docker-compose==1.7.0
-docker-py==1.8.0
-Jinja2==2.8
-PyYAML==3.11
-requests==2.7.0
+docker-compose>=1.7.0
+Jinja2>=2.8
+PyYAML>=3.11
```

Comment:
@dustymab, open an issue to track the need for an RPM and the work done to date. When the time is right, we'll add it to a roadmap. Roadmap docs live [here](https://github.com/ansible/ansible-container/tree/develop/docs/rst/roadmaps).